packages/cli/src/templates/state.tpl

```diff
@@ -0,0 +1,20 @@
+import { State, Action, StateContext } from '@ngxs/store';
+import { {{pascalCase name}}Action } from './{{dashCase name}}.actions';
+
+export class {{pascalCase name}}StateModel {
+  public items: string[];
+}
+
+@State<{{pascalCase name}}StateModel>({
+  name: '{{camelCase name}}',
+  defaults: {
+    items: []
+  }
+})
+export class {{pascalCase name}}State {
+  @Action({{pascalCase name}}Action)
+  add(ctx: StateContext<{{pascalCase name}}StateModel>, action) {
```

Comment:
Add type for action:
`action: {{pascalCase name}}Action`

Reply:
done